src/MotionSource/Targets/FemaleTarget.cs

```diff
@@ -0,0 +1,127 @@
+﻿using System.Collections.Generic;
+using ToySerialController.Utils;
+using UnityEngine;
+using Leap.Unity.Infix;
+using Leap.Unity;
+
+namespace ToySerialController.MotionSource
+{
+    public class FemaleTarget : AbstractPersonTarget
+    {
+        protected override DAZCharacterSelector.Gender TargetGender => DAZCharacterSelector.Gender.Female;
+
+        protected override IEnumerable<string> Targets => new List<string>
+        {
+            "Auto", "Vagina", "Pelvis", "Hips", "Anus", "Mouth", "Left Hand", "Right Hand", "Chest", "Left Foot", "Right Foot", "Feet"
+        };
+
+        protected override string DefaultTarget => "Vagina";
+
+        protected override string TargetPersonStorageKey => "Female";
+
+        protected override string TargetPointStorageKey => "FemaleTarget";
+
+        public FemaleTarget() : base()
+        {
+            TargetUpdaters["Vagina"] = UpdateVaginaTarget;
+            TargetUpdaters["Pelvis"] = () => UpdateFreeControllerTarget("pelvisControl");
+            TargetUpdaters["Hips"] = () => UpdateFreeControllerTarget("hipControl");
+            TargetUpdaters["Chest"] = UpdateChestTarget;
+
+            AutoUpdaters.Insert(0, UpdateVaginaTarget);
```

Comment:
Why insert, the algorithm loops thru all of them anyway.

Reply:
Didn't want to change the existing behaviour and the first has priority. It's unlikely to make a difference, though.